pytorch_lightning/plugins/training_type/training_type_plugin.py

```diff
@@ -71,7 +70,11 @@ def accelerator(self, accelerator: "pl.accelerators.accelerator.Accelerator") ->
 
     @property
     def checkpoint_io(self) -> CheckpointIO:
-        return self._checkpoint_io
+        return self._checkpoint_io if self._checkpoint_io is not None else TorchCheckpointIO()
```

Comment:
quick ques: why return `TorchCheckpointIO` here if there is no checkpoint_io?

Reply:
I'm following the same pattern used below for `precision_plugin`